src/components/AttendeeTable/index.tsx

```diff
@@ -0,0 +1,98 @@
+'use client';
+
+import { UserNamesAndIds } from '@/lib/prisma/users';
+import Paper from '@mui/material/Paper';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableFooter from '@mui/material/TableFooter';
+import TableHead from '@mui/material/TableHead';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+import Typography from '@mui/material/Typography';
+import { useState } from 'react';
+
+interface Props {
+  attendees: UserNamesAndIds | null;
+  noAttendeesText: string;
+}
+
+export default function AttendeeTable({ attendees, noAttendeesText }: Props) {
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+
+  if (!attendees) return null;
```

Comment:
Should there be some kind of error indicating missing attendees? 
```suggestion
  if (!attendees) return <Alert severity="error">Error fetching attendees...</Alert>
```

Reply:
The attendees array is null if the user is a trainee, in which case the attendee table should not be rendered.

With that said, I now realise that this is a relic from an earlier version, where the attendee list component was rendered below the course description in the Course modal (and was hidden from trainees). In the current setup the AttendeeTable component is rendered only if the Course modal view state is set to 'attendees' - which only an admin or a trainer can do. So if attendees ends up being null in the AttendeeTable, something has in fact gone wrong.

Reply:
I'll go with null for now.